coderd/audit.go

```diff
@@ -147,6 +148,9 @@ func (api *API) generateFakeAuditLog(rw http.ResponseWriter, r *http.Request) {
 	if params.Time.IsZero() {
 		params.Time = time.Now()
 	}
+	if params.AdditionalFields == nil {
```

Comment:
This will not catch empty `[]byte{}` which will fail to unmarshal. Check for length instead.
```suggestion
	if len(params.AdditionalFields) == 0 {
```